packages/ag-charts-website/src/features/api-documentation/apiReferenceHelpers.ts

```diff
@@ -252,9 +252,15 @@ export function getNavigationDataFromPath([basePath, ...path]: NavigationPath[],
         if (specialType?.[item.type] === 'InterfaceArray') {
             const child = path.shift();
             if (child) {
-                data.pathname += `${item.name}/${child.name}/`;
+                if (data.hash.startsWith('reference-AgChartOptions-')) {
+                    const prePath = data.hash.slice(25).split('-').filter(Boolean).concat(item.name);
```

Comment:
Nit: Maybe extract `'reference-AgChartOptions-'` to a constant and then use `.length` to avoid trivial find/replace mishaps?